qabel_provider/models.py

```diff
@@ -21,9 +36,18 @@ class Profile(models.Model, ExportModelOperationsMixin('profile')):
     plus_notification_mail = models.BooleanField(default=False)
     pro_notification_mail = models.BooleanField(default=False)
 
-    block_quota = models.BigIntegerField(verbose_name='block server quota (in bytes)', default=2 * 1024**3)
-    monthly_traffic_quota = models.BigIntegerField(verbose_name='block server traffic quota per month (in bytes)',
-                                                   default=20 * 1024**3)
+    # A user may only be subscribed to one plan at a time.
+    # Note that subscriptions are managed outside qabel-accounting. If the subscription state
+    # changes an appropriate update request is sent to the plan/subscription API.
+    # The default value is constructed during the 0014_add_plans database migration.
+    subscribed_plan = models.ForeignKey(Plan, default='free')
+
+    @property
+    def plan(self):
```

Comment:
since this possibly mutates the object it shouldn't be a property, rather a function. Note mutation in docstring.